GOVERNANCE.md

```diff
@@ -0,0 +1,36 @@
+# Strimzi Governance
+
+This document defines project governance for the Strimzi project.
+
+## Voting
+
+The Strimzi project employs voting to ensure no single member can dominate the project. Any maintainer may cast a vote.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR.
+Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
```

Comment:
How do we can define "suitable"? Isn't it too vague? At same time does any other project define a time for that?

Reply:
I think we can specify that they vote by comment. You might not need time limit for this as it is not lazy vote ... you need 2/3s of maintainers to say "yes", if that takes 1 hour or 1 year might formally not make a difference.

Reply:
Agree, so we should avoid to mention the "time" because there is actually no time, it's just a matter of votes.

Reply:
I guess anyway the sensible thing would be to close the vote if you don't get the votes in reasonable time. Not keep it open for months or years :-).